chart/templates/marquez/deployment.yaml

```diff
@@ -59,7 +59,10 @@ spec:
             - name: POSTGRES_USER
               value: {{ .Values.marquez.db.user }}
             - name: POSTGRES_PASSWORD
-              value: {{ .Values.marquez.db.password }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "marquez.secretName" . }}
+                  key: marquez-password
```

Comment:
Mind if we use the name `marquez-db-password` to make it clear it's for the database?